commands/check.go

```diff
@@ -25,6 +25,9 @@ var CheckSlackCmd = &cobra.Command{
 func init() {
 	CheckSlackCmd.Flags().StringP("file", "f", "", "the Slack export file to transform")
 	CheckSlackCmd.Flags().Bool("debug", true, "Whether to show debug logs or not")
+	CheckSlackCmd.Flags().Bool("skip-email", false, "Ignore empty email addresses from the import file. Note that this results in invalid data.")
```

Comment:
What is the use case for ignoring empty emails and ending up with an invalid import?

Reply:
@wiggin77 This is to help figure out which users failed to migrate. I suppose I was thinking a customer could grep through the output file and search for `"email":""` to see which users have the empty email address, though I suppose they could do that with the input too. I figured there should be an alternative that fits "I don't want the script to bail, and I don't have an email domain to auto-assign the users".